src/components/LoginForm/index.js

```diff
@@ -1,63 +1,33 @@
-import { Button, Grid } from "@material-ui/core";
-import { makeStyles } from "@material-ui/core/styles";
+import {
+  Button,
+  Typography,
+  Grid,
+  TextField,
+  InputAdornment,
+} from "@material-ui/core";
 import { useSession, signIn } from "next-auth/react";
+import Image from "next/image";
 import Router from "next/router";
 import PropTypes from "prop-types";
 import React, { useEffect } from "react";
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    flexGrow: 1,
-    color: "white",
-    background: "none",
-    height: "100vh",
-    margin: "auto",
-  },
-  loginButton: {
-    marginBottom: "1rem",
-    width: "100%",
-    color: "white",
-    "&:hover": {
-      color: "#000",
-    },
-    backgroundColor: theme.palette.primary.main,
-    fontWeight: 800,
-    fontSize: theme.typography.subtitle2.fontSize,
-    height: "3rem",
-    [theme.breakpoints.up("xl")]: {
-      fontSize: theme.typography.subtitle1.fontSize,
-      height: "3.5rem",
-      paddingLeft: "2rem",
-      paddingRight: "2rem",
-    },
-  },
-  buttonContainer: {
-    paddingTop: "1rem",
-    width: "400px",
-    [theme.breakpoints.between("xs", "xs")]: {
-      width: "95vw",
-    },
-    "& .MuiLink-underlineHover": {
-      "&:hover": {
-        textDecoration: "none",
-      },
-    },
-  },
-  formStyles: {
-    display: "flex",
-    flexDirection: "column",
-    alignItems: "center",
-  },
-  item: {
-    color: "black",
-    height: "100vh",
-    display: "flex",
-    justifyContent: "center",
-    alignItems: "center",
-  },
-}));
+import useStyles from "./useStyles";
 
-function Login({ providers: providersProp, ...props }) {
+import Link from "@/twoopstracker/components/Link";
+import Section from "@/twoopstracker/components/Section";
+
+function Login({
+  providers: providersProp,
+  title,
+  description,
+  password,
```

Comment:
the prop `password` is confusing and may cause conflict with password textField.  Can we replace it with `forgotPasswordLabel` or something along that as it is just a text for forgot password prompt